include/grpc/impl/codegen/grpc_types.h

```diff
@@ -373,6 +373,9 @@ typedef struct {
   "grpc.disable_client_authority_filter"
 /** If set to zero, disables use of http proxies. Enabled by default. */
 #define GRPC_ARG_ENABLE_HTTP_PROXY "grpc.enable_http_proxy"
+/** Channel arg to set http proxy per channel. If set, the channel arg
+ *  value will be prefered over the http_proxy envrionment settings. */
```

Comment:
Let's just say "preferred over the environment variable settings", since there is more than one environment variable that we may look at.

Reply:
Modified comment to suggested text.